### DISCUSSION TITLE
Complete the quiz of the Dying Light 2 Tower Raid

### DISCUSSION THREAD

**Discussion by woctezuma** [chr233/ASFEnhance#271] [杂谈 / General]
- https://store.steampowered.com/sale/dyinglight2towerraid
- https://store.steampowered.com/points/shop/app/3059450
- https://www.steamcardexchange.net/index.php?gamepage-appid-3059450
- https://steamdb.info/app/3059450/communityitems/

Command:
```
!dl2 ASF
```

> The Tower Raid Open Beta is here until July 18!
> But what weapons should you take with you?
> Answer these five questions and we'll give you the answer!

![Picture](https://i.imgur.com/WoySEcQ.png)

**Comment by chr233** [comment]
added in 2.1.9.0

**Reply by woctezuma** [reply]
Thank you!
